src/data/builder-tools.js

```diff
@@ -1023,6 +1023,16 @@ export const Showcases = [
       "https://anastasia-labs.github.io/lucid-evolution/documentation/core-concepts/instantiate-evolution",
     tags: ["typescript", "javascript", "sdk", "plutus"],
   },
+  {
+    title: "Weld Wallet Connector",
+    description:
+      "Manage wallet connections across multiple blockchains using a single intuitive interface",
+    preview: require("./builder-tools/weld.png"),
+    website: "https://dev.ada-anvil.io/",
```

Comment:
... In that review I did have to drill down into these doc pages & wonder if either would be a better landing page for builders coming for the first time:
* https://dev.ada-anvil.io/guides/transaction/create-basic-transaction/nextjs-with-weld
* https://dev.ada-anvil.io/guides/smart-contract/escrow/wallet-integration

Reply:
Apparent inconsistency would be resolved by changing the landing page to the repository README itself, as per https://github.com/cardano-foundation/developer-portal/pull/1550#issuecomment-3044879668 = https://github.com/Cardano-Forge/weld

Reply:
```suggestion
    website: "https://github.com/Cardano-Forge/weld/",
```
i.e. @0xBora @rdlrt @Dodilanne how about just changing this?

Reply:
Website as well as getstarted link I suppose?